__tests__/app.test.js

```diff
@@ -247,5 +247,82 @@ describe("/api/articles", () => {
           expect(body.msg).toBe("username not found");
         });
     });
+    test("PATCH: 200 - increments the votes on the specified article", () => {
+      const voteUpdate = {
+        inc_votes: 1,
+      };
+
+      return request(app)
+        .patch("/api/articles/1")
+        .send(voteUpdate)
+        .expect(200)
+        .then((response) => {
+          const { updatedArticle } = response.body;
+          expect(updatedArticle).toMatchObject({
+            article_id: 1,
+            votes: 101,
+            title: "Living in the shadow of a great man",
+            body: "I find this existence challenging",
+            author: "butter_bridge",
+            created_at: "2020-07-09T20:11:00.000Z",
+          });
+          expect(updatedArticle.votes).toBe(100 + 1);
+        })
+        .then(() => {
+          const voteUpdate2 = {
+            inc_votes: -100,
+          };
+          return request(app)
+            .patch("/api/articles/1")
+            .send(voteUpdate2)
+            .expect(200)
+            .then((response) => {
+              const { updatedArticle } = response.body;
+              expect(updatedArticle).toMatchObject({
+                article_id: 1,
+                votes: 1,
+                title: "Living in the shadow of a great man",
+                body: "I find this existence challenging",
+                author: "butter_bridge",
+                created_at: "2020-07-09T20:11:00.000Z",
+              });
+              expect(updatedArticle.votes).toBe(101 - 100);
+            });
+        });
+    });
+    test("PATCH: 404 - responds with an error when an article patch is attempted on an article_id that does not exist", () => {
+      const voteUpdate = {
+        inc_votes: 1,
+      };
+      return request(app)
+        .patch("/api/articles/9999")
+        .send(voteUpdate)
+        .expect(404)
+        .then((response) => {
+          expect(response.body.msg).toEqual("article does not exist");
+        });
+    });
+    test("PATCH: 400 - responds with an error when an article patch is attempted with an invalid article_id ", () => {
+      const voteUpdate = {
+        inc_votes: "test string",
+      };
+      return request(app)
+        .patch("/api/articles/1")
```

Comment:
I think you've gotten a little muddled here. This test is a valid one, but the description is for a different (but also valid) test. The article_id refers to the end of our endpoint e.g. /api/articles/hellothere. I recommend leaving the test as it is and changing the description, and then writing another test for an invalid article_id